File: sbab-assignment-fe/src/components/BusLine.js
const BusLine = ({ busLine }) => {
  return (
    <div>
        <h3>{busLine.busLine}</h3> 
        <ol>
          {busLine.listOfStopPoints.map((busStop, index) => (
            <li>{busStop}</li>
          ))}
        </ol>
    </div>
  )
}

export default BusLine

File: sbab-assignment-be/src/main/java/com/sbab/assignment/domain/BusLine.java
package com.sbab.assignment.domain;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class BusLine {

    private String busLine;
    private List<String> listOfStopPoints;

    public BusLine(String busLineNumber, List<String> listOfStopPoints) {
        this.busLine = busLineNumber;
        this.listOfStopPoints = listOfStopPoints;
    }
}
